source/GameAction.cpp

```diff
@@ -127,6 +127,14 @@ namespace {
 			message += "flagship.";
 		Messages::Add(message, Messages::Importance::High);
 	}
+
+	// Sets a bonus between 1.01 and 2. for the size of the
+	// outfit delivery. Ensures the player makes a profit on
+	// the purchase of outfits.
+	double OutfitBulkBonus(int count)
+	{
+		return count > 500 ? 2. : 1. + count * 0.01;
```

Comment:
Count 501 would return 2.
Count 500 would return 1 + (500 * 0.01 = 5) = 6, exceeding the intended max. bonus of 2 in the summary above.

Changing 'count > 500' to 'count >= 100' would instead cap the bonus at 2.

(Note the >= 100 instead of > 100, since if 'count' is 100, both options would return 2, but the former saves some calculations.)